rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java

```diff
@@ -384,7 +387,7 @@ public String eth_gasPrice() {
 
     @Override
     public String eth_blockNumber() {
-        Blockchain blockchain = worldManager.getBlockchain();
+        Blockchain blockchain = Web3Impl.this.blockchain;
```

Comment:
Can't this assignment be removed? (there are a few along this class)

Reply:
For sure!